pkg/rbac/userData.go

```diff
@@ -23,7 +23,7 @@ const impersonationConfigCreationerror = "Error creating clientset with imperson
 
 // Contains data about the resources the user is allowed to access.
 type UserDataCache struct {
-	userData UserData
+	UserData
```

Comment:
**Reviewers:** This is the root change. 
This brings all the objects in `UserData` and makes them available at `UserDataCache` without having to nest.